os_intfs.c:1003:22: error: initialization from incompatible pointer type

Trying to build on BeagleBone Black:

```
ubuntu@arm:~$ uname -a
Linux arm 4.4.9-ti-r25 #1 SMP Fri May 6 10:28:56 UTC 2016 armv7l armv7l armv7l GNU/Linux
```

It's failing with:

```
  CC [M]  /var/lib/dkms/8192cu/1.10/build/os_dep/linux/os_intfs.o
/var/lib/dkms/8192cu/1.10/build/os_dep/linux/os_intfs.c:1003:22: error: initialization from incompatible pointer type [-Werror=incompatible-pointer-types]
  .ndo_select_queue = rtw_select_queue,
                      ^
/var/lib/dkms/8192cu/1.10/build/os_dep/linux/os_intfs.c:1003:22: note: (near initialization for ‘rtw_netdev_ops.ndo_select_queue’)
cc1: some warnings being treated as errors
scripts/Makefile.build:258: recipe for target '/var/lib/dkms/8192cu/1.10/build/os_dep/linux/os_intfs.o' failed
make[1]: *** [/var/lib/dkms/8192cu/1.10/build/os_dep/linux/os_intfs.o] Error 1
Makefile:1387: recipe for target '_module_/var/lib/dkms/8192cu/1.10/build' failed
make: *** [_module_/var/lib/dkms/8192cu/1.10/build] Error 2
make: Leaving directory '/usr/src/linux-headers-4.4.9-ti-r25'
```

--- a/os_dep/linux/os_intfs.c
+++ b/os_dep/linux/os_intfs.c
@@ -949,7 +949,13 @@ unsigned int rtw_classify8021d(struct sk_buff *skb)
 	return dscp >> 5;
 }
 
+#if (LINUX_VERSION_CODE>=KERNEL_VERSION(3,14,0))
+static u16 rtw_select_queue(struct net_device *dev, struct sk_buff *skb,
+			    void *accel_priv,
+			    select_queue_fallback_t fallback)
+#else
 static u16 rtw_select_queue(struct net_device *dev, struct sk_buff *skb)
+#endif
 {
 	_adapter	*padapter = rtw_netdev_priv(dev);
 	struct mlme_priv *pmlmepriv = &padapter->mlmepriv;

